api/EventType.json

```diff
@@ -0,0 +1,104 @@
+{
+  "api": {
+    "EventType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Events",
+        "support": {
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          },
+          "samsunginternet_android": {
+            "version_added": null
+          },
+          "webview_android": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "click": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Events/click",
+        "support": {
```

Comment:
The contents of `click` must be also contained inside a `__compat` block, that’s why [the build is failing](https://travis-ci.org/mdn/browser-compat-data/builds/370212795#L1098).